forest/shared/src/cli/config.rs

```diff
@@ -66,30 +66,27 @@ impl Default for MainnetSnapshotFetchConfig {
         Self {
             /// Default `mainnet` snapshot URL. The assumption is that it will redirect once and will contain a
             /// `sha256sum` file with the same URL (but different extension).
-            snapshot_url: Url::try_from("https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car").unwrap(),
+            snapshot_url: Url::try_from("https://snapshots.mainnet.filops.net/minimal/latest")
+                .unwrap(),
         }
     }
 }
 
 #[derive(Serialize, Deserialize, PartialEq, Eq)]
 pub struct CalibnetSnapshotFetchConfig {
-    pub snapshot_spaces_url: Url,
-    pub bucket_name: String,
-    pub region: String,
-    pub path: String,
+    pub snapshot_url: Url,
 }
 
 impl Default for CalibnetSnapshotFetchConfig {
     fn default() -> Self {
         // unfallible unwrap as we know that the value is correct
         Self {
-            snapshot_spaces_url: Url::try_from(
-                "https://forest-snapshots.fra1.digitaloceanspaces.com",
+            /// Default `calibrationnet` snapshot URL. The assumption is that it will redirect once and will contain a
+            /// `sha256sum` file with the same URL (but different extension).
+            snapshot_url: Url::try_from(
+                "https://snapshots.calibrationnet.filops.net/minimal/latest",
```

Comment:
We would prefer to use our own snapshots.

Reply:
It would be nice to have a flag somewhere though, by default use our own infra but be able to download snapshots from other trusted sources as well, in case we are down.

Reply:
@sudo-shashank Looks like you inverted columns in your table, no?

Reply:
corrected